client/my-sites/plans/plan-overview/plan-remove/index.jsx

```diff
@@ -0,0 +1,120 @@
+/**
+ * External dependencies
+ */
+import Dispatcher from 'dispatcher';
+import page from 'page';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import Dialog from 'components/dialog';
+import { isInGracePeriod } from 'lib/plans';
+import notices from 'notices';
+import paths from '../../paths';
+import { fetchSitePlansCompleted } from 'state/sites/plans/actions';
+import wpcom from 'lib/wp';
+
+const PlanRemove = React.createClass( {
+	propTypes: {
+		plan: React.PropTypes.object.isRequired,
+		selectedSite: React.PropTypes.oneOfType( [
+			React.PropTypes.object,
+			React.PropTypes.bool
+		] ).isRequired,
+		store: React.PropTypes.object.isRequired
+	},
+
+	getInitialState() {
+		return {
+			showDialog: false
+		};
+	},
+
+	removePlan( closeDialog ) {
+		wpcom.undocumented().cancelPlanTrial( this.props.plan.id, ( error, data ) => {
+			closeDialog();
+
+			if ( data && data.success ) {
+				this.props.store.dispatch( fetchSitePlansCompleted( this.props.selectedSite.ID, data.plans ) );
+
+				Dispatcher.handleViewAction( {
+					type: 'FETCH_SITES'
+				} );
+
+				page( paths.plansDestination( this.props.selectedSite.slug, 'free-trial-canceled' ) );
+			} else {
+				notices.error( error.message || this.translate( 'There was a problem removing the plan. Please try again later or contact support.' ) );
+			}
+		} );
+	},
+
+	closeDialog( action ) {
+		this.setState( { showDialog: false } );
+	},
+
+	showDialog( event ) {
+		event.preventDefault();
+
+		this.setState( { showDialog: true } );
+	},
+
+	renderCard() {
+		return (
+			<CompactCard className="plan-remove">
+				<strong>{ this.translate( 'Not looking to purchase?' ) }</strong>
+				{ ' ' }
+				{ this.translate( 'No problem, you can remove the plan and all its features from your site.' ) }
+				{ ' ' }
+				<a href="#" onClick={ this.showDialog }>{ this.translate( 'Remove now.' ) }</a>
```

Comment:
This might be a bit opinionated but I'd remove the period here. In essence it's a button label and usually they don't have sentence punctuation.